base/server/src/main/java/org/dogtagpki/server/rest/SystemConfigService.java

```diff
@@ -270,7 +272,7 @@ public SystemCertData createRequest(CertificateSetupRequest request) throws Exce
                     usagesMask = CryptoUtil.ECDHE_USAGES_MASK;
                 }
 
-                keyPair = CryptoUtil.generateECCKeyPair(token, curveName, usages, usagesMask);
+                keyPair = nssdb.createECKeyPair(token, curveName, usages, usages);
```

Comment:
Actually, shouldn't this be `usagesMask`?

Reply:
Nice catch! Fixed.